src/site/includes/metatags.drupal.liquid

```diff
@@ -0,0 +1,71 @@
+<link rel="canonical" href="{{ hostUrl }}{{ entityUrl.path }}" />
+{% if changed %}
+<meta name="DC.Date" content="{{changed| dateFromUnix: 'YYYY-MM-DD'}}" property="http://purl.org/dc/terms/date">
+{% endif %}
+{% assign tagsSize = entityMetatags | size %}
+{% if tagsSize > 0 %}
+  <!-- Metatags -->
+  {% for tag in entityMetatags %}
+    {% if tag.__typename == "MetaValue" %}
+<meta name="{{ tag.key }}" content="{{ tag.value }}">
+    {% else %}
+      {% if tag.__typename == "MetaProperty" %}
+<meta property="{{ tag.key }}" content="{{ tag.value }}">
+      {% else %}
+        {% if tag.__typename == "MetaLink" %}
+<link rel="{{ tag.key }}" href="{{ tag.value }}">
+        {% endif %}
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+{% else %}
```

Comment:
I could be reading this wrong, but this means that if Drupal has provided `entityMetatags` (such as if the content author manually entered metadata), then we should render those in a loop? But otherwise, we fall back to the meta tags below?

Reply:
That's right. Pages that come directly from drupal pages have entityMetatags, which are generated by a drupal module, but pages that we're creating in metalsmith_drupal (e.g. listing pages) don't.